### ISSUE TITLE
Accessibility should be a core design value

### ISSUE THREAD

**Issue by capnmidnight** [capnmidnight/Primrose#119]
The Primrose website almost certainly does not conform to accessibility guidelines for enabling differently-abled people to use the site and tools within.

The W3C provides a convenient list of tools. This is probably a good starting point: https://www.w3.org/WAI/ER/tools/

A lot more research will have to go into this problem.

It's also an issue with the Framework itself, one that once properly addressed will benefit everyone. Accessibility is largely a design issue, and usage of apps on Google Cardboard vs. HTC Vive mirror many of the same accessibility issues that people of different capabilities experience in real life. We should also not assume that, just because a person is using a full VR system like the Vive, that they have the physical capacity to use it to its full capacity.

Thus, Primrose should have built within it a means of expressing User Interfaces by their actionable interfaces and provide standardize realizations of those actions that fit varying levels of interaction capability. We should be able to provide Vive users with a fully immersive experience, but also a fall-back interface for anyone who cares to trigger it.
